### PR TITLE
feat(lanes/loadtime): include lane in reports

### PR DESCRIPTION
**Pull request by hvanz**
Depends on #3825 

---

#### PR checklist

- [ ] Tests written/updated
- [ ] Changelog entry added in `.changelog` (we use [unclog](https://github.com/informalsystems/unclog) to manage our changelog)
- [ ] Updated relevant documentation (`docs/` or `spec/`) and code comments

### CODE REVIEW

**Review by sergio-mena — APPROVED** [review]
🚀

File: scripts/qa/reporting/latency_plotter.py [hunk 1]
```diff
@@ -44,6 +44,7 @@ def plot_all_experiments(release, csv):
         # Group by connection number and transaction rate
         paramGroups = group.groupby(['connections','rate'])
         for (subKey) in paramGroups.groups.keys():
+
```
Comment (sergio-mena) [inline scripts/qa/reporting/latency_plotter.py:47]:
```suggestion
```

File: scripts/qa/reporting/latency_plotter.py [hunk 2]
```diff
@@ -52,7 +53,7 @@ def plot_all_experiments(release, csv):
             localStartTime = tz.localize(datetime.fromtimestamp(startTime)).astimezone(pytz.utc)
             localEndTime  = tz.localize(datetime.fromtimestamp(endTime)).astimezone(pytz.utc)
             print('experiment', key ,'start', localStartTime.strftime("%Y-%m-%dT%H:%M:%SZ"), 'end', localEndTime.strftime("%Y-%m-%dT%H:%M:%SZ"), 'duration', endTime - startTime, "mean", mean)
-
+
```
Comment (sergio-mena) [inline scripts/qa/reporting/latency_plotter.py:56]:
```suggestion
```

File: scripts/qa/reporting/latency_throughput.py [hunk 3]
```diff
@@ -127,6 +241,7 @@ def compute_experiments_stats(experiments):
     conns = sorted(stats.keys())
     avg_latencies = {}
     throughput_rates = {}
+
```
Comment (sergio-mena) [inline scripts/qa/reporting/latency_throughput.py:244]:
```suggestion
```

**Review by sergio-mena — APPROVED** [review]
LGTM

**Comment by hvanz** [comment]
Maybe change the title to something more generic? (Now it is "Rotating Nodes".)

**Pull request merged**